upgrades/delete_dashboards.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+
+import ConfigParser
+import requests
+
+from requests.auth import HTTPBasicAuth
+
+
+def delete_dashboards(server_ip, user, password):
+
+    dashboards = ["cluster-dashboard", "brick-dashboard", "host-dashboard",
+                  "volume-dashboard"]
+
+    headers = {'content-type': 'application/json'}
+
+    for dashboard in dashboards:
+        url = "http://%s:3000/api/dashboards/db/%s" % (server_ip, dashboard)
+        print (url)
+        response = requests.delete(url, headers=headers,
+                                   auth=HTTPBasicAuth(user, password))
+        if response.status_code == 200:
+            print "Deleted", dashboard, "\n "
+        else:
+            print "Failed to delete", dashboard, response.json(), "\n"
+
+
+    # Deleting the alerts dashboards
+    url = "http://%s:3000/api/orgs/name/Alert_dashboard" \
+          % server_ip
+    print "Getting alerts organization id\n",url,"\n"
+    response = requests.get(url, headers=headers,
+                            auth=HTTPBasicAuth(user, password))
+    resp = response.json()
+
+    if 'id' in resp:
+        id = resp['id']
+        url = "http://%s:3000/api/orgs/%s" % (server_ip, id)
+        print "Deleting alerts organization\n", url
+        response = requests.delete(url, headers=headers,
+                                   auth=HTTPBasicAuth(user, password))
+        resp = response.json()
+        if resp == {u'message': u'Organization deleted'}:
+            print "Deleted Alert dashboards"
+        else:
+            print "Failed to delete Alert dashboards ", resp
+
+    else:
+        print "Failed to delete Alert dashboards."
+        print "Organization id not found", resp
+
+
+def main():
+
+    # getting grafana admin_username and password
+    config = ConfigParser.ConfigParser()
+    config.read('/etc/tendrl/monitoring-integration/grafana/grafana.ini')
+    default_user = config.get('security', 'admin_user')
+    default_password = config.get('security', 'admin_password')
+    default_ip = "127.0.0.1"
+
+    username = raw_input("Please enter the grafana admin_username "
+                         "default-[%s]: " % default_user) or default_user
+    password = raw_input("Please enter the grafana admin_password default"
+                         "-[%s]: " % default_password) or default_password
+
+    print "\n Clearing grafana dashboards \n"
+    delete_dashboards(server_ip=default_ip, user=username,
+                      password=password)
+    print "\n Complete -- Please start tendrl-monitoring-integration service"
```

Comment:
And carbon-cache service?

Reply:
But in command line as well we should to restart tendrl-monitoring-integration and carbon-cache and order should be start carbon-cache and then tendrl-monitoring-integration I feel.

@nthomas-redhat what you say? 

Otherwise code look good to me.

Reply:
@shtripat We will not stop carbon cache as there is no interference with it. Should we stop carbon-cache ?

Reply:
In no interference then its fine